setup.py

```diff
@@ -74,8 +74,13 @@ class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
     c_opts = {
         'msvc': ['/EHsc', '/openmp', '/O2'],
-        'unix': ['-O3', '-march=native'],  # , '-w'
+        #'unix': ['-O3', '-march=native'],  # , '-w'
+        'unix': ['-O3'],  # , '-w'
     }
+    if not os.environ.get("HNSWLIB_NO_NATIVE"):
+        if "conda" not in sys.version.lower():
```

Comment:
A quick question here - why not anaconda? 
It seems like it cancels the use of AVX for all anaconda python builds.

Reply:
I think you're right.  Originally I was trying to catch when it was building the binary in the conda-forge systems, but with the flag now, that isn't necessary.  I'll remove the "conda" check and resubmit for review.